80s Game/Assets/Scripts/Multiplayer/PlayerInputWrapper.cs

```diff
@@ -184,6 +195,17 @@ public void Update()
             }
         }
 
+        //Increase delay between shots
+        fireDelay += Time.deltaTime;
+
+        //If fire button is held and the delay is exceeded
+        if (!fireHeld && fireDelay >= 0.2f)
```

Comment:
Might wanna have two variables for tracking the delay, one for the timer and another for the reset so we dont have to use hard coded values here.